trepb/readme.md

```diff
@@ -12,17 +12,56 @@ Os dados devem estar de acordo com a [Resolução 102 do CNJ](https://atos.cnj.j
 
 ## Como usar
 
+  
+### Rodando sem uso do docker:
 - É preciso ter o compilador de Go instalado em sua máquina. Mais informações [aqui](https://golang.org/dl/).
+
 - Rode o comando abaixo, com mês, ano, cpf e nome que você quer ter acesso as informações
-- Um arquivo .env com nome e cpf deve ser preenchido na pasta trepb
-	- O nome deve ser completo. O cpf deve ter o formato xxx.xxx.xxx-xx. Informações para obter chave de acesso para a API.
+
+- Um arquivo .env.example na pasta raíz indica as variáveis de ambiente que precisam ser passadas para o coletor. 
+	- As informações de NAME e CPF são necessárias para obter uma chave de acesso à api.
+	- O nome deve ser completo. O cpf deve ter o formato xxx.xxx.xxx-xx.
+- O resultado do coletor, CrawlingResult, possui um campo que indica o commit do git usado para dar o build. Para que ele seja setado adequadamente, é precisso passar o commit como argumento do build.
+ 
 
 ```sh
 cd crawler/trepb
-go build
-./trepb --mes=${MES} --ano=${ANO}"
+go get
+go build -ldflags "-X main.gitCommit=$(git rev-parse -1 HEAD)"
+./trepb --mes=${MES} --ano=${ANO}
```

Comment:
Você ainda precisa exportar as variáveis no arquivo `.env`, certo?

https://stackoverflow.com/questions/19331497/set-environment-variables-from-file-of-key-value-pairs/19331521

Reply:
O próprio coletor lê o .env na pasta raiz. Mas também é possível exportar as variáveis antes ou na linha em que o comando é chamado:

Exemplo: NAME="Joao Silva" CPF="123.456.789-00" ./trepb --mes=2 --ano=2019

Reply:
Eita .. eu achava que você tinha removido essa parte que lê o env. Talvez tenha lido errado.